lib/ex_machina/sequence.ex

```
@@ -3,6 +3,12 @@ defmodule ExMachina.Sequence do
     Agent.start_link(fn -> HashDict.new end, name: __MODULE__)
   end
 
+  def next(sequence_name) do
```

Should we raise a helpful error for non-string values?